src/inverseRobot.js

```diff
@@ -7,7 +7,18 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+
+  for (const key in robot) {
+    const newKey = robot[key];
+
+    if (newRobot[newKey]) {
```

Comment:
[CODE STYLE] - use correct check for property presence in object. Some properties can be present, but still contain falsy value.

GOOD EXAMPLE:

if (key in robot) {
EVEN BETTER EXAMPLE:

if (robot.hasOwnProperty(key)) {

BAD EXAMPLE:

if (robot[key] === undefined) {

if (robot[key]) {